components/kyma-environment-broker/internal/avs/delegator.go

```diff
@@ -34,15 +27,15 @@ func NewDelegator(avsConfig Config, operationsStorage storage.Operations) *Deleg
 	}
 }
 
-func (del *Delegator) DoRun(logger logrus.FieldLogger, operation internal.ProvisioningOperation, modelSupplier func(provisioningOperation internal.ProvisioningOperation) (*BasicEvaluationCreateRequest, error)) (internal.ProvisioningOperation, time.Duration, error) {
+func (del *Delegator) DoRun(logger logrus.FieldLogger, operation internal.ProvisioningOperation, evalAssistant EvalAssistant, url string) (internal.ProvisioningOperation, time.Duration, error) {
 	logger.Infof("starting the step")
 
-	if operation.AvsEvaluationInternalId != 0 {
+	if evalAssistant.CheckIfAlreadyDone(operation) {
 		msg := fmt.Sprintf("step has already been finished previously")
 		return del.operationManager.OperationSucceeded(operation, msg)
```

Comment:
please change this line to:
```
return operation, 0, nil
```
to skip this step.

Currently the
```
OperationSucceeded(operation, msg)
```
ends the whole provisioning process with success

Reply:
don't mind, we fix it in this PR: https://github.com/kyma-incubator/compass/pull/1019

Reply:
please return `return operation, 0, nil` here

Reply:
ok, from the previous comment, it seems I do not need to change it.